src/pcapi/core/offerers/api.py

```diff
@@ -82,6 +84,24 @@ def update_venue(
     return venue
 
 
+def upsert_venue_contact(venue: Venue, contact_data: venues_serialize.EditVenueContactBodyModel) -> models.VenueContact:
+    """
+    Create and attach a VenueContact to a Venue if it has none.
+    Update (replace) an existing VenueContact otherwise.
+    """
+    venue_contact = venue.contact
+    if not venue_contact:
+        venue_contact = models.VenueContact(venue=venue)
+
+    venue_contact.email = contact_data.email
+    venue_contact.phone_number = contact_data.phone_number
+    venue_contact.social_medias = contact_data.social_medias
```

Comment:
```suggestion
    venue_contact.social_medias = contact_data.social_medias or {}
```
Ce permettra d'éviter d'avoir des None dans social_medias

Reply:
ou alors:
```suggestion
    if contact_data.social_medias:
        venue_contact.social_medias = contact_data.social_medias
```